vue-frontend/src/components/jobs-thankyou/LatestBlog.vue

```diff
@@ -0,0 +1,202 @@
+<template>
+  <section class="tw-container tw-mb-12 md:tw-mb-36">
+    <div
+      class="tw-flex tw-flex-col tw-items-center tw-mt-24 tw-mb-7 tw-m-auto tw-w-full"
+    >
+      <h2
+        class="tw-text-[1.875rem] md:tw-text-[3.4375rem] tw-leading-[2.8125rem] md:tw-leading-[5.15625rem] tw-text-center tw-text-black-core/[0.87] tw-font-inter-bold"
+      >
+        Latest Blogs
+      </h2>
+    </div>
+    <div
+      class="tw-block lg:tw-hidden tw-flex tw-flex-col md:tw-flex-row md:tw-mt-[4rem] lg:tw-min-h-[580px] lg:tw-pb-[15%] lg:tw-py-0"
+    >
+      <div
+        class="tw-relative tw-flex tw-flex-col tw-items-center md:tw-items-start md:tw-justify-center md:tw-flex-[60%] md:tw-ml-[5%] lg:tw-ml-[10%] xl:tw-ml-[15%]"
+      >
+        <div
+          v-for="(blog, index) in blogs"
+          :key="blog"
+          @mouseover="activeIndex = index"
+          @mouseleave="activeIndex = index"
+          @touchstart.passive="activeIndex = index"
+          @click.native="mixpanel.track('tap_blog_post')"
+          class="tw-block tw-mb-[20px] tw-w-full tw-border tw-border-black-core/[0.6] tw-rounded-[20px] tw-box-border tw-p-4"
+        >
+          <div class="tw-flex tw-flex-col tw-w-full">
+            <div
+              @click="openBlog(blog)"
+              class="tw-block tw-mt-[1rem] tw-w-full"
+            >
+              <img
+                :src="blog.thumbnail"
+                class="tw-object-cover tw-rounded-lg tw-cursor-pointer"
+                loading="lazy"
+                :alt="blog.title"
+              />
+            </div>
+            <div
+              class="tw-flex tw-flex-row tw-justify-between tw-my-[10px] tw-font-inter-regular tw-text-[0.875rem] tw-leading-[1.3125rem] tw-text-black-core/[0.6]"
+            >
+              <span class="tw-w-auto">
+                {{ blog.author }}
+              </span>
+            </div>
+            <div class="tw-my-[10px]">
+              <span
+                @click="openBlog(blog)"
+                class="tw-font-inter-semibold tw-text-[1.25rem] tw-leading-[1.875rem] tw-text-black-core[0.87] tw-cursor-pointer"
+              >
+                {{ blog.title }}</span
+              >
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div
+      class="tw-hidden lg:tw-block tw-flex tw-flex-col md:tw-flex-row md:tw-mt-[4rem] lg:tw-min-h-[580px] lg:tw-py-0"
```

Comment:
Same as above comment

Reply:
here block will display in lg.

Reply:
It will not display in `md`. And you attached CSS of `md` devices